doc/introduction/index.rst

```diff
@@ -429,6 +441,9 @@ The Boot process proceeds as follows:
    the ACRN Device Model and Virtual bootloader through dm-verity
 #. Virtual bootloader starts the User-side verified boot process
 
+In this boot mode, the boot options are defined via the ``VM{x}_CONFIG_OS_BOOTARGS``
+macro in the source code (replace ``{x}`` with the VM number).
+
```

Comment:
This depends, for SOS_VM and PRE_LAUNCHED_VM, the boot options are defined via the ``VM{x}_CONFIG_OS_BOOTARGS`` macro, but for POST_LAUNCHED_VM, the boot option is defined in UOS launch script if bootloader is VSBL, or in the .config file which embedded in disk image if bootloader is OVMF.

Reply:
Suggest we only mention Service VM boot here , we need have other words for User VM in other PRs.

Reply:
> for POST_LAUNCHED_VM, the boot option is defined in UOS launch script if bootloader is VSBL

Good point, I forgot about VSBL. Let me see how I can add some text to cover this (at least mention it)

> or in the .config file which embedded in disk image if bootloader is OVMF.

I guess that's one way someone can change the kernel. But I did not see any specific kernel parameter set in there, can you point these out to me?

We also already provide instructions on how to modify the kernel command-line parameters in various User VMs by modifying the User OS bootloader (it being grub, systemd-boot) directly - see https://projectacrn.github.io/latest/tutorials/running_ubun_as_user_vm.html#enable-the-ubuntu-console-instead-of-the-user-interface for example. I personally like it better because it's more flexible and easier than having to change the kernel config, rebuild it and update the image with these new kernel and modules. So I'd suggest we only focus on that method here, is that OK with you?

Reply:
For Ubuntu/Debian User VMs, we just modify the kernel cmdline in grub, but for ClearImage, the cmdline is located in /loader/entries/acrn.conf of EFI partition (need to mount the img to a loop device).
Or shall we mention it in https://projectacrn.github.io/latest/tutorials/building_uos_from_clearlinux.html#build-uos-image-in-clear-linux-os ?

Reply:
> For Ubuntu/Debian User VMs, we just modify the kernel cmdline in grub, but for ClearImage, the cmdline is located in /loader/entries/acrn.conf of EFI partition (need to mount the img to a loop device).

Yes. That config file is how  one can configure `systemd-bootd`, which is the EFI bootloader used by Clear Linux. So here as well, the method boils down to knowing what EFI bootloader your OS uses, and configure it according to your liking.

> Or shall we mention it in https://projectacrn.github.io/latest/tutorials/building_uos_from_clearlinux.html#build-uos-image-in-clear-linux-os ?

It would be good to mention it in there, but I still want to have the overall mechanism described in a central place.